testing/dart_isolate_runner.h

```diff
@@ -0,0 +1,198 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
This appears to be a a big refactor.  It would be easier to review and manage if this was split into a different PR.

Reply:
I'll split this into a separate PR.

Reply:
https://github.com/flutter/engine/pull/16933